doc/releases/reef.rst

```diff
@@ -6,6 +6,432 @@ Reef is the 18th stable release of Ceph. It is named after the reef squid
 (Sepioteuthis).
 
 
+v18.2.4 Reef
+============
+
+This is the fourth backport release in the Reef series. We recommend that all users update to this release.
+
+An early build of this release was accidentally exposed and packaged as 18.2.3 by the Debian project in April.
+That 18.2.3 release should not be used. The official release was re-tagged as v18.2.4 to avoid
+further confusion.
+
+Notable Changes
+---------------
+
+* RADOS: This release fixes a bug (https://tracker.ceph.com/issues/61948) where pre-reef clients were allowed
+  to connect to the `pg-upmap-primary` (https://docs.ceph.com/en/reef/rados/operations/read-balancer/)
+  interface despite users having set `require-min-compat-client=reef`, leading to an assert in the osds
+  and mons. You are susceptible to this bug in reef versions prior to 18.2.3 if 1) you are using an osdmap
+  generated via the offline osdmaptool with the `--read` option or 2) you have explicitly generated pg-upmap-primary
+  mappings with the CLI command. Please note that the fix is minimal and does not address corner cases such as
+  adding a mapping in the middle of an upgrade or in a partially upgraded cluster (related trackers linked
+  in https://tracker.ceph.com/issues/61948). As such, we recommend removing any existing pg-upmap-primary
+  mappings until remaining issues are addressed in future point releases.
+  See https://tracker.ceph.com/issues/61948#note-32 for instructions on how to remove existing
+  pg-upmap-primary mappings.
+* RBD: When diffing against the beginning of time (`fromsnapname == NULL`) in
+  fast-diff mode (`whole_object == true` with `fast-diff` image feature enabled
+  and valid), diff-iterate is now guaranteed to execute locally if exclusive
+  lock is available.  This brings a dramatic performance improvement for QEMU
+  live disk synchronization and backup use cases.
+* RADOS: `get_pool_is_selfmanaged_snaps_mode` C++ API has been deprecated
+  due to being prone to false negative results.  Its safer replacement is
+  `pool_is_in_selfmanaged_snaps_mode`.
+* RBD: The option ``--image-id`` has been added to `rbd children` CLI command,
+  so it can be run for images in the trash.
+
+Changelog
```

Comment:
Also noting that the changelog is missing CentOS-related changes appended after the last release attempt on May 30.  This is everything under "Commits on Jul 8, 2024" in https://github.com/ceph/ceph/pull/57873/commits -- e.g. https://github.com/ceph/ceph/pull/56019 for the first two commits ("qa: use allowlist instead of whitelist for selinux denials" and "qa: ignore container checkpoint/restore related selinux denials for centos9"), https://github.com/ceph/ceph/pull/57932 for many of the following commits, etc.